app/(main)/account/settings/page.tsx

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+'use client';
+import GlobalSpinner from '@/components/GlobalSpinner/GlobalSpinner';
+import Heading from '@/components/Heading/Heading';
+import LinkCustom from '@/components/LinkCustom/LinkCustom';
+import ResetPasswordForm from '@/components/RestPasswordForm/ResetPasswordForm';
+import UpdateEmailForm from '@/components/UpdateEmailForm/UpdateEmailForm';
+import { useAuthContext } from '@/context/AuthContextProvider';
+import { ChevronLeft } from 'lucide-react';
+import { JSX, useEffect, useState } from 'react';
+
+/**
+ * Display user preferences
+ * @returns {JSX.Element} The rendered user preferences component.
+ */
+const AccountSettings = (): JSX.Element => {
+  const [loadingData, setLoadingData] = useState<boolean>(true);
+  const { isSignedIn } = useAuthContext();
+
+  useEffect(() => {
+    if (isSignedIn) {
+      setLoadingData(false);
+    }
+  }, [isSignedIn]);
+
+  return (
+    <>
+      {loadingData ? (
+        <GlobalSpinner />
+      ) : (
+        <section className="mx-auto max-w-5xl pt-10">
+          <header
+            className="flex flex-col gap-4"
+            data-testid="settings-page-header"
+          >
+            <div data-testid="entry-page-header-to-leagues-link">
```

Comment:
nit: the naming convention of this data-testid is a bit confusing, could be more clear

Reply:
This is mine from a previous PR, @choir27

Reply:
@ryandotfurrer @choir27 updated this to "link-to-all-leagues-page"